adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -296,7 +303,18 @@ public CalendarDate addCalendarDate(@NotNull final CalendarDate newCalendarDate)
      */
     @Override
     public CalendarDate getCalendarDateByServiceIdDate(final String serviceId, final LocalDate date) {
```

Comment:
Is this method used? If not you could remove it for now - it's not clear to me that it would be used in practice (unless I'm missing a use case).

Reply:
I think we tried to anticipate usage but I also lean towards extending APIs as needs arise and rely on your experience.

Reply:
I would say remove for now.